pkg/cmd/user.go

```diff
@@ -100,12 +100,16 @@ func getPasswordFromFlagOrInput() (pw string) {
 }
 
 func getUserFromArg(s *xorm.Session, arg string) *user.User {
+	filter := user.User{}
```

Comment:
Please rename this to `u` instead

Reply:
rename filter to u? `u` is also the return value and response from GetUserWithEmail.

Reply:
Didn't pay attention to that, makes sense. Please keep it.